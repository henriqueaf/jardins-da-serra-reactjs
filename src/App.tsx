import React from 'react';
import './App.css';
// import Routes from './routes';
import Map from './pages/Map';

function App() {
  return <Map />;
}

export default App;
