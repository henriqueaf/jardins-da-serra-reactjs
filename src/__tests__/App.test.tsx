import React from 'react';
import { render } from '@testing-library/react';
import App from '../App';

describe('App component', () => {
  const mockGeolocation = {
    watchPosition: jest.fn(),
    clearWatch: jest.fn(),
  };

  beforeAll(() => {
    // @ts-ignore #2540
    global.navigator.geolocation = mockGeolocation;
  });

  const build = () => {
    const wrapper = render(<App />);
    return {
      wrapper,
    };
  };

  it('renders app component', () => {
    const { wrapper } = build();
    expect(wrapper).toMatchSnapshot();
  });
});
