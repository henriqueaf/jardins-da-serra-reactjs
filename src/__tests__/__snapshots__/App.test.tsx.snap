// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders app component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="map"
      >
        <div
          class="mapFooter"
        >
          <div
            class="currentPositionInfo"
          >
            Você está: undefined / undefined
          </div>
        </div>
        <div
          class="leaflet-container leaflet-grab leaflet-touch-drag"
          style="position: relative;"
          tabindex="0"
        >
          <div
            class="leaflet-pane leaflet-map-pane"
            style="left: 0px; top: 0px;"
          >
            <div
              class="leaflet-pane leaflet-tile-pane"
            >
              <div
                class="leaflet-layer "
                style="z-index: 1;"
              >
                <div
                  class="leaflet-tile-container leaflet-zoom-animated"
                  style="z-index: 18; left: 0px; top: 0px;"
                >
                  <img
                    alt=""
                    class="leaflet-tile"
                    role="presentation"
                    src="https://c.tile.openstreetmap.org/18/131071/131071.png"
                    style="width: 256px; height: 256px; left: -256px; top: -256px;"
                  />
                  <img
                    alt=""
                    class="leaflet-tile"
                    role="presentation"
                    src="https://a.tile.openstreetmap.org/18/131072/131071.png"
                    style="width: 256px; height: 256px; left: 0px; top: -256px;"
                  />
                  <img
                    alt=""
                    class="leaflet-tile"
                    role="presentation"
                    src="https://a.tile.openstreetmap.org/18/131071/131072.png"
                    style="width: 256px; height: 256px; left: -256px; top: 0px;"
                  />
                  <img
                    alt=""
                    class="leaflet-tile"
                    role="presentation"
                    src="https://b.tile.openstreetmap.org/18/131072/131072.png"
                    style="width: 256px; height: 256px; left: 0px; top: 0px;"
                  />
                </div>
              </div>
            </div>
            <div
              class="leaflet-pane leaflet-shadow-pane"
            >
              <img
                alt=""
                class="leaflet-marker-shadow leaflet-zoom-hide"
                src="marker-shadow.png"
                style="margin-left: -12px; margin-top: -41px; width: 41px; height: 41px; left: 0px; top: 0px;"
              />
            </div>
            <div
              class="leaflet-pane leaflet-overlay-pane"
            />
            <div
              class="leaflet-pane leaflet-marker-pane"
            >
              <img
                alt=""
                class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
                src="marker-icon.png"
                style="margin-left: -12px; margin-top: -41px; width: 25px; height: 41px; left: 0px; top: 0px; z-index: 0;"
                tabindex="0"
              />
            </div>
            <div
              class="leaflet-pane leaflet-tooltip-pane"
            />
            <div
              class="leaflet-pane leaflet-popup-pane"
            />
          </div>
          <div
            class="leaflet-control-container"
          >
            <div
              class="leaflet-top leaflet-left"
            >
              <div
                class="leaflet-control-zoom leaflet-bar leaflet-control"
              >
                <a
                  aria-label="Zoom in"
                  class="leaflet-control-zoom-in leaflet-disabled"
                  href="#"
                  role="button"
                  title="Zoom in"
                >
                  +
                </a>
                <a
                  aria-label="Zoom out"
                  class="leaflet-control-zoom-out"
                  href="#"
                  role="button"
                  title="Zoom out"
                >
                  −
                </a>
              </div>
            </div>
            <div
              class="leaflet-top leaflet-right"
            />
            <div
              class="leaflet-bottom leaflet-left"
            />
            <div
              class="leaflet-bottom leaflet-right"
            >
              <div
                class="leaflet-control-attribution leaflet-control"
              >
                <a
                  href="https://leafletjs.com"
                  title="A JS library for interactive maps"
                >
                  Leaflet
                </a>
                 | © 
                <a
                  href="http://osm.org/copyright"
                >
                  OpenStreetMap
                </a>
                 contributors
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="map"
    >
      <div
        class="mapFooter"
      >
        <div
          class="currentPositionInfo"
        >
          Você está: undefined / undefined
        </div>
      </div>
      <div
        class="leaflet-container leaflet-grab leaflet-touch-drag"
        style="position: relative;"
        tabindex="0"
      >
        <div
          class="leaflet-pane leaflet-map-pane"
          style="left: 0px; top: 0px;"
        >
          <div
            class="leaflet-pane leaflet-tile-pane"
          >
            <div
              class="leaflet-layer "
              style="z-index: 1;"
            >
              <div
                class="leaflet-tile-container leaflet-zoom-animated"
                style="z-index: 18; left: 0px; top: 0px;"
              >
                <img
                  alt=""
                  class="leaflet-tile"
                  role="presentation"
                  src="https://c.tile.openstreetmap.org/18/131071/131071.png"
                  style="width: 256px; height: 256px; left: -256px; top: -256px;"
                />
                <img
                  alt=""
                  class="leaflet-tile"
                  role="presentation"
                  src="https://a.tile.openstreetmap.org/18/131072/131071.png"
                  style="width: 256px; height: 256px; left: 0px; top: -256px;"
                />
                <img
                  alt=""
                  class="leaflet-tile"
                  role="presentation"
                  src="https://a.tile.openstreetmap.org/18/131071/131072.png"
                  style="width: 256px; height: 256px; left: -256px; top: 0px;"
                />
                <img
                  alt=""
                  class="leaflet-tile"
                  role="presentation"
                  src="https://b.tile.openstreetmap.org/18/131072/131072.png"
                  style="width: 256px; height: 256px; left: 0px; top: 0px;"
                />
              </div>
            </div>
          </div>
          <div
            class="leaflet-pane leaflet-shadow-pane"
          >
            <img
              alt=""
              class="leaflet-marker-shadow leaflet-zoom-hide"
              src="marker-shadow.png"
              style="margin-left: -12px; margin-top: -41px; width: 41px; height: 41px; left: 0px; top: 0px;"
            />
          </div>
          <div
            class="leaflet-pane leaflet-overlay-pane"
          />
          <div
            class="leaflet-pane leaflet-marker-pane"
          >
            <img
              alt=""
              class="leaflet-marker-icon leaflet-zoom-hide leaflet-interactive"
              src="marker-icon.png"
              style="margin-left: -12px; margin-top: -41px; width: 25px; height: 41px; left: 0px; top: 0px; z-index: 0;"
              tabindex="0"
            />
          </div>
          <div
            class="leaflet-pane leaflet-tooltip-pane"
          />
          <div
            class="leaflet-pane leaflet-popup-pane"
          />
        </div>
        <div
          class="leaflet-control-container"
        >
          <div
            class="leaflet-top leaflet-left"
          >
            <div
              class="leaflet-control-zoom leaflet-bar leaflet-control"
            >
              <a
                aria-label="Zoom in"
                class="leaflet-control-zoom-in leaflet-disabled"
                href="#"
                role="button"
                title="Zoom in"
              >
                +
              </a>
              <a
                aria-label="Zoom out"
                class="leaflet-control-zoom-out"
                href="#"
                role="button"
                title="Zoom out"
              >
                −
              </a>
            </div>
          </div>
          <div
            class="leaflet-top leaflet-right"
          />
          <div
            class="leaflet-bottom leaflet-left"
          />
          <div
            class="leaflet-bottom leaflet-right"
          >
            <div
              class="leaflet-control-attribution leaflet-control"
            >
              <a
                href="https://leafletjs.com"
                title="A JS library for interactive maps"
              >
                Leaflet
              </a>
               | © 
              <a
                href="http://osm.org/copyright"
              >
                OpenStreetMap
              </a>
               contributors
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
